modules/microshift-custom-ca-con.adoc

```diff
@@ -4,42 +4,43 @@
+When {microshift-short} starts, an internal {microshift-short} cluster certificate authority (CA) issues the default API server certificate. By default, clients outside of the cluster cannot verify the {microshift-short}-issued API server certificate. To grant access and encrypt connections between the {microshift-short} API server and external clients, you can replace the default certificate with a custom server certificate that is issued externally by a CA that clients trust.
 
-. Copy the certificates and keys to the preferred directory in the host operating system. Ensure that the files are accessible by root only.
+//Q: are these steps still valid?
+
+The following steps illustrate the workflow for customizing the API server certificate configuration in {microshift-short}:
+
+. Copy the certificates and keys to the preferred directory in the host operating system. Ensure that the files are accessible only with root access.
 
 . Update the {microshift-short} configuration for each custom CA by specifying the certificate names and new fully qualified domain name (FQDN) in the {microshift-short} `/etc/microshift/config.yaml` configuration file.
 +
 Each certificate configuration can contain the following values:
 
-** The certificate file location is a required value.
-** A single common name containing the API server DNS and IP address or IP address range.
+* The certificate file location is a required value.
+* A single common name containing the API server DNS and IP address or IP address range.
 +
 --
 [TIP]
 ====
 In most cases, {microshift-short} generates a new `kubeconfig` for your custom CA that includes the IP address or range that you specify. The exception is when wildcards are specified for the IP address. In this case, {microshift-short} generates a `kubeconfig` with the public IP address of the server. To use wildcards, you must update the `kubeconfig` file with your specific details.
 ====
 --
-** Multiple Subject Alternative Names (SANs) containing the API server DNS and IP addresses or a wildcard certificate.
-** You can provide additional DNS names for each certificate.
+* Multiple Subject Alternative Names (SANs) containing the API server DNS and IP addresses or a wildcard certificate.
+* You can provide additional DNS names for each certificate.
 
 . After the {microshift-short} service restarts, you must copy the generated `kubeconfig` files to the client.
 
 . Configure additional CAs on the client system. For example, you can update CA bundles in the {op-system-base-full} truststore.
++
+[IMPORTANT]
+====
+Custom server certificates must be validated against CA data configured in the trust root of the host operating system. For information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/using-shared-system-certificates_securing-networks#the-system-wide-trust-store_using-shared-system-certificates[The system-wide truststore].
```

Comment:
```suggestion
Custom server certificates must be validated against CA data configured in the trust root of the host operating system. For information, see the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/using-shared-system-certificates_securing-networks#the-system-wide-trust-store_using-shared-system-certificates[system-wide truststore].
```
Just cosmetic (and I read that avoiding filling words in links titles helps indexing).